_data/categories.yml

```
@@ -1,42 +1,45 @@
+監視/可観測性: 🕵️
+CDN/セキュリティ: 🛡️
+ビジネス/業務ツール: 💼
+勤怠管理/人事労務: 💼
+ワークフロー自動化: ⚡
+データ分析: 📊
+デザインツール: 🎨
+動画/メディア: 🎥
+デスクトップ/GUIエージェント: 🖱️
+マルチエージェント基盤: 🕸️
+AI画像/メディア生成: 🎨
+AIコードレビュー: 👀
```

YAMLlint (1.38.0)

[error] duplication of key "AIコードレビュー" in mapping

(key-duplicates)